src/app/helptext/system/failover.ts

```diff
@@ -23,17 +24,26 @@ export const helptext_system_failover = {
     sync_to_message: T("Sync to peer succeeded."),
   },
 
-  disabled_placeholder: T('Disabled'),
-  disabled_tooltip: T(''),
+  disabled_placeholder: T('Disable Failover'),
+  disabled_tooltip: T('Disables automatic failover and changes the \
```

Comment:
Use present tense.  Also... what the icon says is not needed.  It's a side effect of disabling failover, not something the user changes directly.
```suggestion
  disabled_tooltip: T('Disable automatic failover.'),
```